index.html

```diff
@@ -60,11 +60,37 @@
             </div>
             <div class="info">
                 <div class="info__margin">
-                    <!-- Consider updating the UI to display flags instead of text -->
-                    [<a href="#" title="English" data-lang="en">en</a>]
-                    [<a href="#" title="Spanish" data-lang="es">es</a>]
-                    [<a href="#" title="French"  data-lang="fr">fr</a>]
-                    [<a href="#" title="Chinese" data-lang="zh">zh</a>]
+
+
+                    <div id="lang-dropdown">
+                      <img id="currentLanguage" class="iconflag" src="">
+                      <div id="lang-dropdown-content">
+                        <!-- "Language supported" in the same order as they appear in Microsoft's product page:
+                             https://www.microsoft.com/en-us/p/age-of-empires-ii-definitive-edition/9njdd0jgpp2q -->
```

Comment:
From the Microsoft website:

![image](https://user-images.githubusercontent.com/2985436/79060518-d31c0e00-7c3a-11ea-99d9-0e6becada234.png)